build-logic/src/main/kotlin/nessie-conventions-root.gradle.kts

```diff
@@ -38,26 +38,31 @@ val projectName = rootProject.file("ide-name.txt").readText().trim()
 val ideName = "$projectName ${rootProject.version.toString().replace(Regex("^([0-9.]+).*"), "$1")}"
 
 if (System.getProperty("idea.sync.active").toBoolean()) {
+
   idea {
     module {
       name = ideName
       isDownloadSources = true // this is the default BTW
       inheritOutputDirs = true
 
-      val sparkScalaProps = Properties()
-      val integrationsDir = projectDir.resolve("integrations")
+      val nessieRootProjectDir =
```

Comment:
Could you maybe add a comment here explaining the logic? Otherwise it's a bit intriguing.